apps/astarte_appengine_api/test/astarte_appengine_api_web/plug/group_name_decoder_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Astarte.AppEngine.APIWeb.Plug.GroupNameDecoderTest do
+  use Astarte.AppEngine.APIWeb.ConnCase
+  use ExUnitProperties
+  alias Astarte.AppEngine.APIWeb.Plug.GroupNameDecoder
+  alias Astarte.AppEngine.API.GroupTestGenerator
+
+  @max_subpath_count 10
+
+  describe "random generated group_name validation" do
+    @tag type: :unit_test
+    @tag issue: 904
+    property "call/2 decode path" do
```

Comment:
`describe` + `test` or `describe` + `property` should read like a phrase. "random generated group_name validation call/2 decode path" does not read like one, should be something like
```elixir
describe "GroupNameDecoder" do
  property "correctly decodes group names" do
    # ...
  end
end
```
A `describe` block is not even stricly necessary here since you only have one test, in that case you can just put all the description in the `property`